.github/workflows/Tests.yml

```
@@ -46,15 +48,66 @@ jobs:
       run: php artisan key:generate --env=testing
     - name: Directory Permissions
       run: chmod -R 777 storage bootstrap/cache
+    - name: Install ChromeDriver
+      run: php artisan dusk:chrome-driver --detect
     - name: Migrate the damn database
       run: php artisan migrate
     - name: Execute tests (Unit and Feature tests) via PHPUnit/Pest
       run: php artisan test --parallel
+    
+    - name: Create Dusk CI Environment
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:2:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)